app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
why adding :color to attr_accessor list when you are proposing it to be a column of the table?